i18n/messages/nl.json

```diff
@@ -572,6 +577,11 @@
     "description": "Label for 'disableAddingRemovingRows' builder field",
     "originalDefault": "Disable adding or removing groups"
   },
+  "NbDr3m": {
+    "defaultMessage": "Geeft gebruikers de mogelijkheid een lijn te tekenen, wanneer ze met het kaartmateriaal werken",
```

Comment:
```suggestion
    "defaultMessage": "Gebruikers kunnen rechte lijnen tekenen op de kaart.",
```

Reply:
is it one or more lines, or just one? Probably good to clarify that in the tooltip.